mono/metadata/threads.c

```diff
@@ -960,7 +960,7 @@ fire_attach_profiler_events (MonoNativeThreadId tid)
 		"Handle Stack"));
 }
 
-static guint32 WINAPI start_wrapper_internal(StartInfo *start_info, gsize *stack_ptr)
+static gulong WINAPI start_wrapper_internal(StartInfo *start_info, gsize *stack_ptr)
```

Comment:
32bits either way, but less ambiguous -- guint32 could be unsigned int, and the correct type is unsigned long